CONTRIBUTING.md

```diff
@@ -76,6 +76,7 @@ Documents are declared in a service declaration file, under the `documents` prop
   "documents": {
     "<document type>": {
       "fetch": "<the URL where the document can be found>",
+      "executeClientScripts": "<a boolean which allow client scripts to be executed before accessing the content>",
```

Comment:
```suggestion
      "executeClientScripts": "<a boolean to execute client-side JavaScript loaded by the document before accessing the content, in case the DOM modifications are needed to access the content; defaults to false (fetch HTML only)>",
```